### PR TITLE
Remove old code

### PR DESCRIPTION
**Pull request by kiootic**

### CODE REVIEW

**Comment by louischan-oursky** [comment]
Target another branch such as `authgear`? `master` should be kept for bug fix.

**Comment by kiootic** [comment]
Oh yes, we should be using the `independence-refactor` branch for the refactor, or do you want to change the name?

**Comment by louischan-oursky** [comment]
Any name is fine.

**Comment by louischan-oursky** [comment]
I just rebase the base branch onto master and we have conflicts :(

**Comment by kiootic** [comment]
Let me rebase...

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          go-version: 1.13.3
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Get dependencies
+        run: make vendor
+      - name: Test
+        run: |
+          make generate
+          go mod tidy
+          (set +e; git status --porcelain | grep '.*'; test $? -eq 1)
```
Comment (louischan-oursky) [inline .github/workflows/test.yaml:30]: Move line 28, 29 and 30 to Makefile as a new target.

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          make lint
+          make test
```
Comment (louischan-oursky) [inline .github/workflows/test.yaml:32]: We should run in this order

- `make lint`
- `make test`
- `make generate; go mod tidy; (grep ...)`

And failing one should not prevent following commands from running. However, any failure should fail the whole job.

**Comment by kiootic** [comment]
I'll add back a Dockerfile and make target, but I think image building should be part of deployment flow, so no need to build in workflow now.

**Pull request merged**